### PR TITLE
chore (users.ts): use resultSelector in mergeMap

### PR DESCRIPTION
**Pull request by SekibOmazic**
Using resultSelector in every mergeMap

### CODE REVIEW

File: src/app/models/users.ts [hunk 1]
```diff
@@ -110,24 +110,24 @@ export class Users {
...
 
     let deletes = this.actions$.filter(action => action.type === DELETE_USER)
       .filter(action => ! action.payload.deleting)
       .do(action => this._store.dispatch({type: DELETING_USER, payload: action.payload}))
       .mergeMap(
         action => this._http.delete(`${this._url}/${action.payload.id}`),
-        action => action.payload.id
-      )
-      .map(userId => ({type: DELETED_USER, payload: userId}));
+        (action, res: Response) => ({type: DELETED_USER, payload: action.payload.id})
```
Comment (meenie) [inline src/app/models/users.ts]: You could remove the response parameter.  :tropical_drink:

**Comment by meenie** [comment]
This looks good! Much cleaner looking :).  Feel free to address my inline comment, or just let me know if I should go ahead and merge this.

**Comment by SekibOmazic** [comment]
Driving ATM. Left second param (Response) to have uniform look&feel. Isn't needed at all.

**Comment by meenie** [comment]
Alright, no worries. I'll merge it in.

Don't email and drive! :P

**Pull request merged**